foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/store/ConfigLoader.java

```diff
@@ -285,10 +285,12 @@ public boolean isFullyLoaded() {
         try {
             Property statusProperty = propertyDao.read(new PropertyKey("load-status", "fully-loaded"));
             return statusProperty != null && BooleanUtils.toBoolean(statusProperty.getValue());
+        } catch (NullPointerException e) {
+            logger.warn("Properties are not loaded previously, continue");
```

Comment:
I: Cleaned up to avoid printing stack trace when the load status fails due to null pointer - this happens everything you deploy the config store for the first time.

Reply:
A: Would prefer just doing a `null` check on the nullable value instead of using exceptions for flow control.

Reply:
Updated `VaultPropertDao` to return null when the property doesn't exist. This mirrors the current logic in `KrauseningPropertyDao`